pkg/controller/kibanasecret/controller.go

```diff
@@ -80,5 +79,5 @@ func (r *ReconcileKibanaSecret) Reconcile(request reconcile.Request) (reconcile.
 
 // handleSecret returns true if meta namespace is "openshift-logging" and name is "kibana" or "kibana-proxy".
 func handleSecret(meta metav1.Object) bool {
-	return meta.GetNamespace() == constants.OpenshiftNS && utils.ContainsString([]string{"kibana", "kibana-proxy"}, meta.GetName())
+	return utils.ContainsString([]string{"kibana", "kibana-proxy"}, meta.GetName())
```

Comment:
No need for a NS check against the cluster request?

Reply:
correct, that was tied to `openshift-logging` what we instead do now is for whatever namespace the event was triggered, we try to make changes against/using that namespace's kibana object